Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -2985,6 +2985,12 @@ bool RenderStyle::shouldPlaceVerticalScrollbarOnLeft() const
     return (!isLeftToRightDirection() && isHorizontalWritingMode()) || writingMode() == WritingMode::RightToLeft;
 }
 
+// ignore non-standard ::-webkit-scrollbar when standard properties are in use
```

Comment:
```suggestion
// Ignore non-standard ::-webkit-scrollbar when standard properties are in use.
```

This is the webkit style for comments